experiments/experiments.go

```diff
@@ -334,3 +334,24 @@ func isSimpleExperiment(experimentType string) bool {
 	}
 	return false
 }
+
+// ExperimentEvent is the playload used by Expose to log whether a user has
+// been exposed to an experimental treatment.
+type ExperimentEvent struct {
+	ID              uuid.UUID
+	CorrelationID   uuid.UUID
+	DeviceID        uuid.UUID
+	Experiment      *ExperimentConfig
+	VariantName     string
+	UserID          string
+	LoggedIn        bool
+	ClientTimestamp int64
+	CookieCreatedAt int64
+	AppName         string
+	EventType       string
+}
```

Comment:
Also I assume on the API level, it's up to the caller to fill in `ExperimentEvent` struct and call `Expose` function, is that correct? Can we add some comments here to indicate which fields are required and which are optional?